### DISCUSSION TITLE
taro官方文档API 【支持情况】显示基本上缺少百度小程序的图标，是不支持吗还是遗漏？

### DISCUSSION THREAD

**Discussion by Vinsurs** [NervJS/taro#12085] [Q & A] [answered]
### 这个特性解决了什么问题？
查看【showLoading】api文档，【支持情况】连百度小程序的图标都没有，是不支持吗？还是遗漏了？但是这个 showLoading是最基本的api呀？

### 这个 API 长什么样？
期望【支持情况】部分能尽可能加上遗漏的部分，不然我真的会以为连showLoading 方法 都没适配百度小程序，更何况其他api

**Comment by ZakaryCode** [comment, marked as answer]
<img width="159" alt="image" src="https://user-images.githubusercontent.com/24262362/180121557-a587ba5c-5ca1-4df0-9107-2f1650e21baf.png">

如图，缺失的平台是没有标注的，灰色模糊的才是不支持。

**Comment by ZakaryCode** [comment]
> 期望【支持情况】部分能尽可能加上遗漏的部分

支持情况皆由社区贡献者标注，欢迎共建
